buildSrc/src/main/kotlin/ProjectExtensions.kt

```diff
@@ -62,7 +62,8 @@ inline fun <reified T : BaseExtension> Project.setupBase(
       freeCompilerArgs += listOf(
         "-progressive",
         "-opt-in=kotlin.RequiresOptIn",
-        "-Xjvm-default=all"
+        "-Xjvm-default=all",
+        "-Xcontext-receivers"
```

Comment:
https://blog.jetbrains.com/kotlin/2022/02/kotlin-1-6-20-m1-released/#prototype-of-context-receivers-for-kotlin-jvm